### DISCUSSION TITLE
Accessing Kopia repository from CLI

### DISCUSSION THREAD

**Discussion by dulismo** [vmware-tanzu/velero#5918] [Community support Q&A] [answered]
Hello,  

I was wondering if I could connect to Kopia repository directly from Kopia CLI in order to benefit from Kopia's features, especially for snapshots download, which I miss from Velero's features.    
I tried the following command :  
```
kopia repository connect s3 --bucket=... --access-key=... --secret-access-key=... --endpoint=... --readonly --prefix=/velero/kopia
```

But got this error :  
```
ERROR error connecting to repository: repository not initialized in the provided storage
```

Is this possible ?

**Comment by dulismo** [comment, marked as answer]
I finally figured out : --prefix needs to be `velero/kopia/NAMESPACE_NAME` : without the initial / & ending with the target namespace name, as Velero initializes a Kopia repository for each namespace.

**Reply by seang96** [reply]
Looks like the prefix is now just kopia/NAMESPACE_NAME now but using the cli command I still get the error.

**Reply by seang96** [reply]
Looks like it's kopia/NAMSEPACE_NAME/

The trailing slash is necessary now.

**Comment by iusergii** [comment]
@Nicky31 Does it require a repo password? Any way to get it?

**Reply by dulismo** [reply]
Can't check it right now, but it should be the default velero password for repos : static-passw0rd

**Reply by iusergii** [reply]
Thank you very much!
Found that helm creates secret `repo-credentials` and it static as you mentioned!
